go.mod

```diff
@@ -14,34 +14,34 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.16.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.8.0
 	github.com/aws/aws-sdk-go-v2/credentials v1.17.47
-	github.com/cyphar/filepath-securejoin v0.3.6
+	github.com/cyphar/filepath-securejoin v0.4.1
 	github.com/dimchansky/utfbom v1.1.1
-	github.com/fluxcd/cli-utils v0.36.0-flux.11
+	github.com/fluxcd/cli-utils v0.36.0-flux.12
 	github.com/fluxcd/kustomize-controller/api v1.4.0
-	github.com/fluxcd/pkg/apis/acl v0.5.0
-	github.com/fluxcd/pkg/apis/event v0.15.0
-	github.com/fluxcd/pkg/apis/kustomize v1.8.0
-	github.com/fluxcd/pkg/apis/meta v1.9.0
-	github.com/fluxcd/pkg/http/fetch v0.14.0
-	github.com/fluxcd/pkg/kustomize v1.15.0
-	github.com/fluxcd/pkg/runtime v0.52.0
-	github.com/fluxcd/pkg/ssa v0.43.0
-	github.com/fluxcd/pkg/tar v0.10.0
-	github.com/fluxcd/pkg/testserver v0.9.0
+	github.com/fluxcd/pkg/apis/acl v0.6.0
+	github.com/fluxcd/pkg/apis/event v0.16.0
+	github.com/fluxcd/pkg/apis/kustomize v1.9.0
+	github.com/fluxcd/pkg/apis/meta v1.10.0
+	github.com/fluxcd/pkg/http/fetch v0.15.0
+	github.com/fluxcd/pkg/kustomize v1.16.0
+	github.com/fluxcd/pkg/runtime v0.53.0
+	github.com/fluxcd/pkg/ssa v0.44.0
+	github.com/fluxcd/pkg/tar v0.11.0
+	github.com/fluxcd/pkg/testserver v0.10.0
 	github.com/fluxcd/source-controller/api v1.4.1
 	github.com/getsops/sops/v3 v3.9.2
 	github.com/hashicorp/vault/api v1.15.0
-	github.com/onsi/gomega v1.36.1
+	github.com/onsi/gomega v1.36.2
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/ory/dockertest/v3 v3.11.0
-	github.com/spf13/pflag v1.0.5
-	golang.org/x/net v0.33.0
-	k8s.io/api v0.32.0
-	k8s.io/apimachinery v0.32.0
-	k8s.io/client-go v0.32.0
+	github.com/spf13/pflag v1.0.6
+	golang.org/x/net v0.34.0
+	k8s.io/api v0.32.1
+	k8s.io/apimachinery v0.32.1
+	k8s.io/client-go v0.32.1
 	k8s.io/utils v0.0.0-20241210054802-24370beab758
-	sigs.k8s.io/controller-runtime v0.19.3
-	sigs.k8s.io/kustomize/api v0.18.0
+	sigs.k8s.io/controller-runtime v0.20.1
+	sigs.k8s.io/kustomize/api v0.19.0
```

Comment:
This is pinned below so you need to bump Kustomize packages there, make sure to update the comment as well

Reply:
fixed, thanks